scripts/hooks/prepare-commit-msg

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+STATUS_GO_VERSION_FILE_IS_BEING_COMMITTED=$(git diff --cached --name-only -r | grep status-go-version.json)
+
+if [ -n "$STATUS_GO_VERSION_FILE_IS_BEING_COMMITTED" ]
+then
+  MERGE_BASE=$(git merge-base -a develop "$(git rev-parse --abbrev-ref HEAD)" develop)
+  GO_COMMIT_USED=$(git show "$MERGE_BASE":status-go-version.json | jq -r '."commit-sha1"')
+
+  GITHUB_LINK="https://github.com/status-im/status-go/compare/$(git rev-parse HEAD)...$GO_COMMIT_USED"
```

Comment:
This `git rev-parse HEAD` doesn't make sense. We're comparing `status-go` commits, not `status-go` commits and `status-react` commits.